bodhi/tests/server/services/test_updates.py

```diff
@@ -322,6 +322,51 @@ def test_new_module_update(self, publish, *args):
         # At the end, ensure that the right kind of package was created.
         self.assertEquals(self.db.query(ModulePackage).count(), 1)
 
+    @unittest.skipIf(six.PY3, 'Not working with Python 3 yet')
+    @mock.patch(**mock_valid_requirements)
+    def test_multiple_builds_of_same_module_stream(self, *args):
+        self.create_release(u'27M')
+
+        res = self.app.post_json('/updates/', self.get_update([u'nodejs-6-20170101',
+                                                               u'nodejs-6-20170202']),
+                                 status=400)
+        assert 'Multiple nodejs:6 builds specified' in res, res
+
+    @unittest.skipIf(six.PY3, 'Not working with Python 3 yet')
+    @mock.patch(**mock_valid_requirements)
+    def test_multiple_builds_of_different_module_stream(self, *args):
+        self.create_release(u'27M')
+
+        res = self.app.post_json('/updates/', self.get_update([u'nodejs-6-20170101',
+                                                               u'nodejs-8-20170202']))
+        res = res.json
+        assert res['request'] == 'testing'
+        assert len(res['builds']) == 2
+        assert res['builds'][0]['type'] == 'module'
+        assert res['builds'][1]['type'] == 'module'
+        assert res['builds'][0]['nvr'] == 'nodejs-6-20170101'
+        assert res['builds'][1]['nvr'] == 'nodejs-8-20170202'
+        assert res['title'] == 'nodejs-6-20170101 nodejs-8-20170202'
+
+        # At the end, ensure that the right kind of packages were created.
+        self.assertEquals(self.db.query(ModulePackage).count(), 2)
+
+    @unittest.skipIf(six.PY3, 'Not working with Python 3 yet')
+    @mock.patch(**mock_valid_requirements)
+    def test_multiple_updates_single_module_steam(self, *args):
+        # Ensure there are no module packages in the DB to begin with.
+        self.assertEquals(self.db.query(ModulePackage).count(), 0)
+        self.create_release(u'27M')
+
+        # First create an update for nodejs:6
+        self.app.post_json('/updates/', self.get_update(u'nodejs-6-20170101'))
+
+        # Next create a second update for nodejs:6
+        self.app.post_json('/updates/', self.get_update(u'nodejs-6-20170202'))
```

Comment:
So this one should have obsoleted the former one, right? How about an assertion that the obsoletion happened? There should be two updates now. The first should be obsolete (and have a comment mentioning the new update), and the second should have a comment about inheriting the notes from the first.